backend/app/routes/admin/deleteAdmin.js

```diff
@@ -0,0 +1,45 @@
+const { default: to } = require("await-to-js");
+const constants = require("../../../constants");
+const { ErrorHandler } = require("../../../helpers/error");
+const Admin = require("../../models/Admin");
+
+module.exports = async (req, res, next) => {
+    const { isSuperAdmin } = res.locals.decode;
```

Comment:
How are you checking for superadmin, i couldn't get. How is it verifying that the person trying to delete is the superadmin?

Reply:
In Auth Middleware, we are storing the jwt-payload into res.locals.decode, and jwt key is generated by three fields name, email and isSuperAdmin. 

isSuperAdmin is true for only one entry which is created on requesting the /createSuperAdmin route and it's value are hardcoded.